### DISCUSSION TITLE
你好，温哥华 | HaoLiu's blog - HaoLiu's life journey

### DISCUSSION THREAD

**Discussion by giscus** [lyleLH/haoliu.me#3] [Announcements]
# 你好，温哥华 | HaoLiu's blog - HaoLiu's life journey

等待了十四个月之后，我顺利落地在YVR机场，开始了我新的生活 ...

https://imhaoliu.com/blog/202411/hello_van

**Comment by lyleLH** [comment]
测试一下评论功能🤲

**Reply by lyleLH** [reply]
great!

**Comment by frank1936** [comment]
Hello Tom，hope you have a great time in Ca

**Reply by lyleLH** [reply]
多谢 Frank
